kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/SchemaManagerTest.java

```diff
@@ -384,6 +385,36 @@ public void FieldsWithUnspecifiedModeShouldNotCauseNpe() {
     testGetAndValidateProposedSchema(schemaManager, existingSchema, expandedSchema, expectedSchema);
   }
 
+  @Test
+  public void testTombstoneRecordWithNullValueSchemaShouldNotCauseNpe() {
+    com.google.cloud.bigquery.Schema existingSchema = com.google.cloud.bigquery.Schema.of(
+        Field.newBuilder("f1", LegacySQLTypeName.BOOLEAN).setMode(Field.Mode.REQUIRED).build()
+    );
+
+    com.google.cloud.bigquery.Schema newSchema = com.google.cloud.bigquery.Schema.of(
+        Field.newBuilder("f1", LegacySQLTypeName.BOOLEAN).setMode(Field.Mode.NULLABLE).build()
+    );
```

Comment:
I don't think it is needed in this test, but to use the existing code it is needed as it is called in `testGetAndValidateProposedSchema`, otherwise there will be a `org.mockito.exceptions.misusing.UnfinishedStubbingException`